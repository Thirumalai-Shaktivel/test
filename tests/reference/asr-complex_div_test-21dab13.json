{
    "basename": "asr-complex_div_test-21dab13",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/complex_div_test.f90",
    "infile_hash": "0bbb7100f12786a5fc5022ab6cfd9f97cc6ede016ed0bdab03eda670",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-complex_div_test-21dab13.stdout",
    "stdout_hash": "15d01bf303b36d9bc6e18ae8cb694b7be2a7394a252f695c01a76110",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}