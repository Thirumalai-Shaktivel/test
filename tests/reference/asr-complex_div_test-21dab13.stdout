(TranslationUnit (SymbolTable 1 {complex2: (Program (SymbolTable 2 {x: (Variable 2 x Local () () Default (Complex 4 []) Source Public Required .false.)}) complex2 [] [(= (Var 2 x) (ComplexConstructor (ConstantReal 3.000000 (Real 4 [])) (ConstantReal 4.000000 (Real 4 [])) (Complex 4 []) (ConstantComplex 3.000000 4.000000 (Complex 4 []))) ()) (= (Var 2 x) (BinOp (Var 2 x) Div (ImplicitCast (ConstantReal 4.000000 (Real 4 [])) RealToComplex (Complex 4 []) (ConstantComplex 4.000000 0.000000 (Complex 4 []))) (Complex 4 []) () ()) ()) (Print () [(Var 2 x)]) (= (Var 2 x) (BinOp (ImplicitCast (ConstantInteger 2 (Integer 4 [])) IntegerToComplex (Complex 4 []) ()) Div (Var 2 x) (Complex 4 []) () ()) ()) (Print () [(Var 2 x)]) (= (Var 2 x) (BinOp (ImplicitCast (ConstantInteger 1 (Integer 4 [])) IntegerToComplex (Complex 4 []) ()) Div (BinOp (Var 2 x) Add (ComplexConstructor (ConstantReal 0.000000 (Real 4 [])) (ConstantReal 3.000000 (Real 4 [])) (Complex 4 []) (ConstantComplex 0.000000 3.000000 (Complex 4 []))) (Complex 4 []) () ()) (Complex 4 []) () ()) ()) (Print () [(Var 2 x)])])}) [])
